libs/island-ui/core/src/lib/DatePicker/DatePicker.tsx

```diff
@@ -143,6 +151,9 @@ export const DatePicker: React.FC<React.PropsWithChildren<DatePickerProps>> = ({
               size={size}
             />
           }
+          timeFormat="HH:mm"
```

Comment:
Are we using the same time format for IS and EN ? Shouldn't this be the `timeFormat` from the shared lib `date.ts` ?